multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1605,12 +1638,22 @@ def evaluate_coco(
                 warn_if_exist=self._warn_if_exist,
             )
         self._save_path = os.path.abspath(os.path.expanduser(self._save_path))
-        cache_path = os.path.join(self._save_path, "object_detection_result_cache.json")
+        cocoeval_cache_path = os.path.join(self._save_path, "object_detection_result_cache.json")
```

Comment:
Because only cocoeval needs to cache the result and compute mAP from the result file. Later we might change to a faster cocoeval toolkit so I just let it cache for now.